README.md

```diff
@@ -23,13 +23,54 @@ be printed to your screen! Amazing, right?
 
 ## Use it!
 
+### As a base image:
+
 You can also use this image as a base image for your Carp-related Docker
 projects. This is as simple as starting your Dockerfile with:
 
 ```
 FROM carplang/carp:latest
 ```
 
+### As a carp builder:
+
+``` bash
+### from this repo root:
+docker run -v $(pwd)/example:/mnt/app \
+           --user 1000:1000 \
+           --rm \
+           carplang/carp:latest \
+           carp -b hello_world.carp
+
+## Then you can run resulting executable from the host(If it's x64glibc Linux)
+./example/out/HelloWorld
+
+### Or with a browser(WASM/ASM.JS):
+
+## Generate web files:
+docker run -v $(pwd)/example/out/:/mnt/app \
+            --rm \
+            -ti \
+            carplang/carp:emcc \
+            emcc main.c \
+                 -s WASM=1 \
+                 -I/opt/carp/core \
+                 -o hello.html \
+                 --emrun
```

Comment:
It seems like there is a missing step of generating the `main.c` file. It’s basically the same as above, but maybe it should be mentioned?

Reply:
@hellerve  It assumes that you first run:
``` bash
docker run -v $(pwd)/example:/mnt/app \
           --user 1000:1000 \
           --rm \
           carplang/carp:latest \
           carp -b hello_world.carp
```

but yeah. It can be clearer :thinking: